### PR TITLE
Docs/did module docs review

### PR DESCRIPTION
**Pull request by noandrea**
This PR is to fix/improve the project documentation

### CODE REVIEW

**Review by barriebyron — COMMENTED** [review]
hi team! lgtm, with exception of the temporary status of the HEXADECIMAL example doc

File: x/did/spec/02_state.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ A verification method has the following fields:
 - `verificationMaterial`: a string that is either   
   - `blockchainAccountID` - a string representing a cosmos based account address
   - `publicKeyHex` - a string representing a public key encoded as a hex string
+  - `publicKeyMultibase` - a string representing a public key encoded according to the Multibase Data Format [Hexadecimal upper-case encoding](https://datatracker.ietf.org/doc/html/draft-multiformats-multibase#appendix-B.1)
```
[the remark this answers is not in the document]
Comment (noandrea) [reply x/did/spec/02_state.md:54]: given how things go, atm this is the best source we have

**Review by barriebyron — APPROVED** [review]
lgtm, did we want to put a calendar event reminder to follow up after the standard expires on Feb 22?

**Comment by noandrea** [comment]
> lgtm, did we want to put a calendar event reminder to follow up after the standard expires on Feb 22?

that's a good idea, I'll do that. thanks @barriebyron !

**Pull request merged**